Assets/MixedRealityToolkit.Tools/MSBuild/Scripts/Utilities.cs

```diff
@@ -439,15 +438,14 @@ public static void DeleteDirectory(string targetDir, bool waitForDirectoryDelete
 
             TryIOWithRetries(() => Directory.Delete(targetDir, false), 2, TimeSpan.FromMilliseconds(100), true);
 
-            if (waitForDirectoryDelete)
+            if (waitForDirectoryDelete && Application.isEditor)
             {
-#if UNITY_EDITOR // Just in case make sure this is forced to be Editor only
+                // Just in case make sure this is forced to be Editor only
```

Comment:
wrong spaces?

Reply:
![image](https://user-images.githubusercontent.com/3580640/65922843-9160ff80-e39b-11e9-994c-101a124b37c5.png)

Matches the line below

Reply:
sorry, meant tabs vs spaces. If they're spaces then ignore

Reply:
I thought when GitHub did this "bold" space it meant tabs:
![image](https://user-images.githubusercontent.com/25975362/65923011-2237db00-e39c-11e9-9083-a724f4e90dfc.png)

Reply:
Yep, they're spaces!

Reply:
Ahh I think in this case it's because I'm replaces characters with spaces